src/Widget.php

```diff
@@ -85,9 +85,6 @@ protected function addOptions(array $options, string $defaultClass): array
         if (isset($options['class'])) {
             $class = $options['class'];
             unset($options['class']);
-            if (is_array($class)) {
```

Comment:
Is it not needed?

Reply:
I think it is not necessary, I have thought and not found a case where the options class is an array.

Reply:
Done revert and add tests.